pkg/api/types.go

```diff
@@ -1440,10 +1440,14 @@ func (a *AgentPoolProfile) IsUbuntuNonVHD() bool {
 }
 
 // GetKubernetesLabels returns a k8s API-compliant labels string for nodes in this profile
-func (a *AgentPoolProfile) GetKubernetesLabels(rg string) string {
+func (a *AgentPoolProfile) GetKubernetesLabels(rg string, deprecated bool) string {
```

Comment:
unrelated to the PR but why is this in types and GetMasterKubernetesLabels in helper?

Reply:
No idea.